discoverable-client/src/main/resources/application.yml

```diff
@@ -79,15 +79,11 @@ apiml:
                     - endpoint: /greeting
                       language: java
                       codeBlock: |
-                          ```
                           System.out.println("Greeting code snippet");
-                          ```
                     - endpoint: /{yourName}/greeting
                       language: java
                       codeBlock: |
-                          ```
                           System.out.println("Your name greeting code snippet");
```

Comment:
This could be a better example of a code snippet

Reply:
Done!